.coveragerc

```diff
@@ -3,3 +3,10 @@ omit =
     # we omit numba code since we cannot measure it
     ngmix/*_nb.py
     ngmix/fastexp.py
```

Comment:
We should be testing this function but I guess this is a numab issue?

Reply:
yes, for the coverage printouts